components/gateway/internal/auditlog/service_test.go

```diff
@@ -225,38 +224,16 @@ func TestAuditlogService_LogConfigurationChange(t *testing.T) {
 }
 func TestSink_ChannelStuck(t *testing.T) {
 	//GIVEN
-	timeout := time.Millisecond * 100
-	ctx, _ := context.WithTimeout(context.TODO(), timeout*2)
-	channelSize := 5
-	messageSize := channelSize + 1
-	chanMsg := make(chan auditlog.Message, channelSize)
-	done := make(chan error)
-	sink := auditlog.NewSink(chanMsg, timeout)
+	chanMsg := make(chan auditlog.Message)
```

Comment:
this test is so simple right now :) Good job. Consider renaming test name, for example: `TestSink_TimeoutOnWrite`

Reply:
Yes,  the the test is shorter.